project/Dependencies.scala

```diff
@@ -224,7 +224,7 @@ object Dependencies {
   )
 
   val workbenchServiceTest: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-service-test" % serviceTestV % "test" classifier "tests" excludeAll (excludeGuava, excludeStatsD)
-  val leonardoClient: ModuleID =  "org.broadinstitute.dsde.workbench" %% "leonardo-client" % "1.3.6-563edbd-SNAP"//"1.3.6-9d5d754"
+  val leonardoClient: ModuleID =  "org.broadinstitute.dsde.workbench" %% "leonardo-client" % "1.3.6-8366c9f-SNAP"
```

Comment:
I updated this am and you should not need the SNAP version anymore

Reply:
Oh yeah sorry I keep forgetting you need that to get the autoPause parameter in the client!